src/applications/gi/utils/render.jsx

```diff
@@ -33,10 +33,17 @@ export const renderPreferredProviderFlag = result => {
   );
 };
 
-export const renderLearnMoreLabel = ({ text, modal, showModal, component }) => (
+export const renderLearnMoreLabel = ({
+  text,
+  modal,
+  learnMoreMessage,
```

Comment:
can we call this something like `ariaLabel`?

Reply:
updated var name to ariaLabel